app/Converters/parser_onnx.py

```diff
@@ -112,6 +151,7 @@ def default(self, obj):
         json.dump(layer_info, f, indent=2, cls=CustomEncoder)
 
     print(f"Модель успешно сохранена в {output_json_path}")
+    print(f"Input shape: {input_info.get('shape', [])}")
```

Comment:
Library code prints to stdout. Prefer returning metadata or using a logger to avoid noisy CLI output.